mne_g3d/viz.py

```diff
@@ -65,27 +73,68 @@ def plot_brain_mesh(rh_vertices=None,
 
     fig = ipv.figure(width=fig_size[0], height=fig_size[1], lighting=True)
 
+    if act_data is not None:
+        cmap = get_mesh_cmap(act_data, cmap_str=cmap_str)
+        # data mapping into [0, 1] interval
+        arr_act_data = np.concatenate(tuple(act_data.values()))
+        dt_min = arr_act_data.min()
+        dt_max = arr_act_data.max()
+
+        out_min = 0
+        out_max = 1
+
+        k = (out_max - out_min) / (dt_max - dt_min)
+        b = out_max - k * dt_max
+
+        rh_act_data = k * act_data.get('rh') + b
+        lh_act_data = k * act_data.get('lh') + b
+
+        rh_act_data[rh_act_data < 0] = 0
+        lh_act_data[lh_act_data < 0] = 0
+
+        rh_act_data[rh_act_data > 1] = 1.0
+        lh_act_data[lh_act_data > 1] = 1.0
+    else:
+        rh_act_data = None
+        lh_act_data = None
+        cmap = None
+
     if (rh_vertices is not None) and (rh_faces is not None):
         if offset is not None:
             rh_vertices = offset_hemi(rh_vertices, 'rh', offset)
 
-        rh_mesh = plot_hemisphere_mesh(rh_vertices, rh_faces, rh_color)
+        rh_mesh, _ = plot_hemisphere_mesh(rh_vertices,
+                                          rh_faces,
+                                          rh_color,
+                                          act_data=rh_act_data,
+                                          cmap=cmap)
 
     if (lh_vertices is not None) and (lh_faces is not None):
         if offset is not None:
             lh_vertices = offset_hemi(lh_vertices, 'lh', offset)
 
-        lh_mesh = plot_hemisphere_mesh(lh_vertices, lh_faces, lh_color)
+        lh_mesh, _ = plot_hemisphere_mesh(lh_vertices,
+                                          lh_faces,
+                                          lh_color,
+                                          act_data=lh_act_data,
+                                          cmap=cmap)
+
+    ipv.style.box_off()
+    ipv.style.axes_off()
+    ipv.style.background_color('black')
 
-    style.use('minimal')
```

Comment:
why this change (I'm not that familiar with `ipyvolume` style)?

Reply:
In the latest ipyvolume (from GitHub) styles configuration  was changed, so I changed it here as well.